tests/integration/features/conversation/add-group.feature

```diff
@@ -0,0 +1,212 @@
+    Given user "participant1" creates room "room" (v4)
+      | roomType | 2 |
+      | source | group |
+      | invite |group1 |
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 3               |
+    And user "participant2" is not member of group "group1"
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+
+  Scenario: User is removed from when their last group which is a member of a chat is removed
+    Given user "participant1" creates room "room" (v4)
+      | roomType | 2 |
+      | source | group |
+      | invite |group1 |
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 3               |
+    And user "participant1" removes group "group1" from room "room" with 200 (v4)
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+
+  Scenario: Moderator is removed from a group which is a member of a chat but stays in the chat
+    Given user "participant1" creates room "room" (v4)
+      | roomType | 2 |
+      | source | group |
+      | invite |group1 |
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 3               |
+    And user "participant1" promotes "participant2" in room "room" with 200 (v4)
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 2               |
+    And user "participant2" is not member of group "group1"
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 2               |
+
+  Scenario: Moderator is removed from a group which is a member of a chat but stays in the chat
```

Comment:
The name is the same as the previous scenario (but in this case the group is removed from the room, not the user from the group).